recipes/Commons/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "iscd-commons" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # there are no tagged releases which so using archives of specific commits
+  url: https://github.com/ISCDtoolbox/Commons/archive/refs/tags/v{{ version }}.zip
+  sha256: 2d6e4f02852146609efbb9288b70ca5b6763e6b2ac61fffc64ebd303a3577896
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('iscd-commons', max_pin='x.x.x') }}
+  skip: true  # [py<3 or win] # Cannot open include file: 'pthread.h': No such file or directory
+  ignore_run_exports_from:
+    - {{ compiler('cxx') }}
+  run_exports:
+    - {{ pin_subpackage('iscd-commons', max_pin='x.x.x') }}
```

```suggestion
build:
  number: 0
  skip: true  # [py<3 or win] # Cannot open include file: 'pthread.h': No such file or directory
  ignore_run_exports_from:
    - {{ compiler('cxx') }}
  run_exports:
    - {{ pin_subpackage('iscd-commons', max_pin='x.x.x') }}
```
Remove duplicate run_export